subTitan.sh

```diff
@@ -108,7 +108,8 @@ sed -i "s/NCORES/$num_cores/g" scripts/bs_Core.sh
 }
 
 prepSubmit() {
-
+    #find git tag
+    gittag=$(git describe --tags)
```

Comment:
wasn't the git tag stuff removed from run titan? does this need to be removed too?